sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.unmask

```diff
@@ -1,4 +1,4 @@
 # Overwrite portage-stable mask. We haven't seen LLVM related problems
 # with rust so far, so keep using 1.70.0.
-~dev-lang/rust-1.80.0
-~virtual/rust-1.80.0
+~dev-lang/rust-1.80.1
+~virtual/rust-1.80.1
```

Comment:
Heh, is this needed? I can see the comment saying about overwriting mask for rust 1.70 because of some LLVM problems. I think this could just go away, unless portage-stable profile is masking it.

Reply:
I don't see a problem with not being on the latest and freshest rust release. We have been so far only because the github action worked that way. :)